spark/src/main/scala/geotrellis/spark/io/slippy/FileSlippyTileReader.scala

```diff
@@ -1,71 +0,0 @@
-/*
- * Copyright 2016 Azavea
```

Comment:
So basically you just removed all the `slippy` readers and writers? Why?
The entire slippy package. 

It can be removed only for a good reason and removal probably means creating a project in `geotrellis-contrib`.

A bit more general question: why have you decided to remove it and to remove in terms of this PR?